packages/block-library/src/columns/edit.js

```diff
@@ -70,6 +73,37 @@ function ColumnsEditContainer( {
 		[ `are-vertically-aligned-${ verticalAlignment }` ]: verticalAlignment,
 	} );
 
+	const { childColumnsAttributes } = useSelect( ( select ) => {
+		const { getBlockOrder, getBlockAttributes } = select( 'core/block-editor' );
+		const childColumns = getBlockOrder( clientId );
+
+		return {
+
+			childColumnsAttributes: childColumns.map( ( block ) => getBlockAttributes( block ) ),
+
+		};
+	}, [ clientId ] );
+
+	let columnsTemplateString = ``;
```

Comment:
This being the string that describes the grid, maybe it would be better-called something with 'grid' in it. Like 'gridValue' or 'gridStructure'? When I read template I think of the HTML of the columns or something ...

Reply:
It's a CSS template 😁 
I'll switch it to the name of the CSS property we're actually using it on, `gridTemplateColumns`. That should make things clearer.